Modules/_decimal/_decimal.c

```diff
@@ -3854,21 +3916,27 @@ dec_as_integer_ratio(PyObject *self, PyObject *Py_UNUSED(dummy))
     return result;
 }
 
+/*[clinic input]
+_decimal.Decimal.to_integral
```

Comment:
Would it be better to have `_decimal.Decimal.to_integral_value` here? `to_integral` is noted as an alias of `to_integral_value`.

Reply:
Now to_integral_value docstring - also moved.